Bugsnag/KSCrash/Source/KSCrash/Recording/Sentry/BSG_KSCrashSentry.h

```diff
@@ -101,6 +105,22 @@ typedef struct BSG_KSCrash_SentryContext {
     /** Length of the stack trace. */
     int stackTraceLength;
 
+    /** All threads at the time of the crash.
+     * Note: This will be a kernel-allocated array that must be manually kernel-deallocated.
+     */
+    thread_t *allThreads;
+    unsigned allThreadsCount;
+
+    // NOTE: Due to a compiler bug, this must NOT be placed after threadsToResume.
+    //       The mach struct is safe to be placed afterwards.
+    unsigned threadsToResumeCount;
```

Comment:
Can you be a bit more specific on the version of iOS affected and a brief explanation of what you saw? Just thinking that this will worrying to our future selves to maintain/refactor if they don't know in what way it was breaking.

Reply:
Nick found the problem. Another test was initialising a global variable from a local stack-allocated one, and later tests were overwriting its data when they reached that point in the stack.